amethyst_rendy/src/lib.rs

```diff
@@ -49,3 +49,9 @@ pub use util::{simple_shader_set, ChangeDetection};
 pub mod loaders {
     pub use rendy::texture::palette::{load_from_linear_rgba, load_from_srgb, load_from_srgba};
 }
+
+// feature = "dx12" is removed here because rendy does not currently support it.
+static_assertions::assert_cfg!(
```

Comment:
Should we warn / error if somebody enables dx12?

Reply:
This technically shoudn't be removed. It's not rendy, but hal that doesn't support command buffers. The support might be added as non-breaking change though, because the api is already defined.

Reply:
Hmm, but right now people can enable the `dx12` feature which leads to reports like #1643

Reply:
I've left this removed for now, since dx12 is not supported (and opengl support is coming quickly).  We can always re-add them when they are supported, but I'd rather steer people correctly for now.